src/test/kotlin/blackjack/domain/ResultTest.kt

```diff
@@ -0,0 +1,71 @@
+package blackjack.domain
+
+import blackjack.domain.card.Card
+import blackjack.domain.card.Denomination
+import blackjack.domain.card.Shape
+import blackjack.domain.state.Bust
+import blackjack.domain.state.FirstTurn
+import io.kotest.matchers.collections.shouldContainExactlyInAnyOrder
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+internal class ResultTest {
+    @Test
+    fun `승리 결과`() {
+        Result.WIN.value shouldBe "승리"
+    }
+
+    @Test
+    fun `패배 결과`() {
+        Result.LOSE.value shouldBe "패배"
+    }
+
+    @Test
+    fun `딜러와 플레이어를 받아 승패를 구분할 수 있다`() {
+        val ten = Card(Shape.DIAMOND, Denomination.TEN)
+        val nine = Card(Shape.DIAMOND, Denomination.NINE)
+        val six = Card(Shape.DIAMOND, Denomination.SIX)
+        val five = Card(Shape.DIAMOND, Denomination.FIVE)
+
+        val dealer = Dealer(FirstTurn.draw(ten, six))
+        val player1 = Player("one", FirstTurn.draw(ten, nine))
+        val player2 = Player("one", FirstTurn.draw(ten, five))
+        val players = listOf(player1, player2)
+
+        Result.dealerResult(dealer, players) shouldContainExactlyInAnyOrder listOf(Result.WIN, Result.LOSE)
+        Result.playerResult(dealer, player1) shouldBe Result.WIN
+        Result.playerResult(dealer, player2) shouldBe Result.LOSE
```

Comment:
```suggestion
        Result.dealerResult(dealer, players) shouldContainExactlyInAnyOrder listOf(Result.WIN, Result.LOSE)
```
shouldContainExactlyInAnyOrder 👍👍

shouldContainExactlyInAnyOrder 만 사용해줘도 괜찮을 것 같아요~

플레이어의 결과이기 때문에 추후 로직 변경으로 인하여 바로 확인이 가능하기 때문이죠